src/pages/announcements/utils.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import yup from '../../utils/yup-config';
+import { Duration, parse, serialize } from 'tinyduration';
+
+export const ID = 'id';
+export const DATE = 'creationDate';
+export const MESSAGE = 'message';
+export const DURATION = 'duration';
+export const DAYS = 'days';
+export const HOURS = 'hours';
+export const MINUTES = 'minutes';
```

Comment:
Can we avoid using constants for all object keys ? To me it's not readable

Reply:
I agree I did like we do everywhere by coherence. How would you do it ?